packages/frontend/src/Components/Search.tsx

```diff
@@ -15,9 +15,12 @@ const Search = () => {
         // TODO: Phase 2: Make the Search API call to our node backend
         // NOT the actual spotify API
         // the node backend will call the spotify API
+        axios.get(`/myApi/search?q=${searchText}`).then((res) => {
```

Comment:
Better and more clear way to write this:
axios
        .get('/myApi/search', {
          params: { ...searchParams },
        })
        .then(({ data }) => {
          console.log(data);
          setSearchResults(data);
          setLoading(false);
        })
2 Notes:
1. {data} instead of res
2. Use params: { ___ }